lago_python_client/models/plan.py

```diff
@@ -1,8 +1,10 @@
 from typing import List, Optional
 
 from lago_python_client.base_model import BaseModel
+from lago_python_client.models.usage_threshold import UsageThreshold
```

Comment:
I guess you could add this import in the `from .usage_threshold` line

Reply:
woops! I'll clean that up.